### ISSUE TITLE
View Pending/Completed/Failed/Submitted Ratio at Orion UI

### ISSUE THREAD

**Issue by BitTheByte** [PrefectHQ/prefect#7123]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar request and didn't find it.
- [X] I searched the Prefect documentation for this feature.

### Prefect Version

2.x

### Describe the current behavior

Not Implemented

### Describe the proposed behavior

It'd be very useful to track and view `Pending/Completed/Failed/Summited` task runs at the UI as an indicator of progress

**Comment by BitTheByte** [comment]
@zhen0 per a flow run since rader view is not very useful at large scales 

Let's assume that I have a flow of 100k task runs of T1 and T2. I'd like to see something like the table below at the flow run page

```
T1: Pending: 9889, Running: 165 Completed: 700, Crashed: 6, Submitted: 1000
T2: Pending: 60500, Running: 234, Completed: 1356, Crashed: 10, Submitted: 56
...
```

Also something like average execution time so far and projected time for task runs would be useful.

I'd don't really know if we should also consider displaying an average execution time of a deployment based on the previous flow runs but I think it's worth the discussion.

Basically, I'd like to see a statistical view of my flow runs and deployment for better discussion making and hunting bugs

**Comment by zhen0** [comment]
Thanks @BitTheByte - I like the idea and am going to flag this one for our product team.

**Comment by WillRaphaelson** [comment]
Thanks @BitTheByte - Will here from Prefect - we're planning on a broader rethinking of our flow run pages later this year and into Q1, so I'd like to park this for now and reach back out when we're doing user research for that project.

But while we're here - I'm somewhat concerned that this table would become unwieldy / awkward if the fan got wider, i.e 100 runs of 10 tasks. What would you think about collapsing this into a single summary statistic row more like a progress bar for all tasks?

**Comment by BitTheByte** [comment]
Hello @WillRaphaelson

Thanks for taking this idea into consideration, using a progress bar will be confusing as task runs are variable so seeing that my tasks reached 50% after some while changing to 30% will confuse new users
